src/Microsoft.OpenApi.OData.Reader/Operation/OperationHandlerProvider.cs

```diff
@@ -68,11 +68,11 @@ public OperationHandlerProvider()
                 {OperationType.Delete, new NavigationPropertyDeleteOperationHandler() }
             };
 
-            // navigatoin property ref (Get/Post/Patch/Delete)
+            // navigati0n property ref (Get/Post/Put/Delete)
             _handlers[ODataPathKind.Ref] = new Dictionary<OperationType, IOperationHandler>
             {
                 {OperationType.Get, new RefGetOperationHandler() },
-                {OperationType.Patch, new RefPatchOperationHandler() },
+                {OperationType.Put, new RefPutOperationHandler() },
                 {OperationType.Post, new RefPostOperationHandler() },
```

Comment:
We should also apply the change to POST. See https://docs.microsoft.com/en-us/graph/api/group-post-members?view=graph-rest-1.0&tabs=http#example-1-add-a-member-to-a-group

Reply:
Resolved